package.json

```diff
@@ -62,14 +63,12 @@
     "hsw-cli": "./bin/hsw-cli"
   },
   "scripts": {
-    "lint": "eslint $(cat .eslintfiles) || exit 0",
-    "lint-ci": "eslint $(cat .eslintfiles)",
+    "lint": "eslint bin/* etc/* .",
```

Comment:
Why moving away from the `.eslintfiles` approach?

Reply:
Actually this was giving trouble. Restored `.eslintfiles` @pinheadmz